src/components/ChatInput/ChatInput.module.scss

```diff
@@ -13,5 +37,10 @@
   .button {
     height: 40px;
     width: 140px;
+    flex-shrink: 0;
+
+    @media (max-width: 480px) {
+      width: 100%;
+    }
```

Comment:
I would prefer that the design is done first for the mobile design, then at the bottom, after all mobile designs, a single @media (min-width: <space>) for tablet/desktop containing all style updates for those sizes? Please make a note in the codermanz agent with this style preference